packages/block-library/src/cover/edit.native.js

```diff
@@ -262,6 +270,42 @@ const Cover = ( {
 		</InspectorControls>
 	);
 
+	const colorPickerControls = (
+		<InspectorControls>
+			<BottomSheetConsumer excludeFromSettings={ true }>
+				{ ( {
+					shouldEnableBottomSheetScroll,
+					shouldDisableBottomSheetMaxHeight,
+					onCloseBottomSheet,
+					isBottomSheetContentScrolling,
+				} ) => (
+					<ColorPicker
+						shouldEnableBottomSheetScroll={
+							shouldEnableBottomSheetScroll
+						}
+						shouldDisableBottomSheetMaxHeight={
+							shouldDisableBottomSheetMaxHeight
+						}
+						setColor={ ( color ) => {
+							setCustomColor( color );
+							setColor( color );
+						} }
+						onNavigationBack={ () => {
+							setCustomColorPickerShowing( false );
+							closeSettingsBottomSheet();
+						} }
+						onCloseBottomSheet={ onCloseBottomSheet }
```

Comment:
When the modal is closed by tapping outside of it the value of `isCustomColorPickerShowing` will remain as `true`. We'd need to pass a closure function with `onCloseBottomSheet` that would handle the logic to set that to `false`.

On Android we would also need to handle `onHardwareButtonPress` when the user dismisses the modal by taping on the hardware back button.

Reply:
If we fix this we could use `isCustomColorPickerShowing` to render the `colorPickerControls` as I explain below.